src/components/GGanttBarTooltip.vue

```diff
@@ -11,7 +11,7 @@
         }"
       >
         <div class="g-gantt-tooltip-color-dot" :style="{ background: dotColor }" />
-        <slot>
+        <slot :bar="bar" :bar-start="barStartRaw" :bar-end="barEndRaw">
```

Comment:
I'm aware that the start and end value is part of the bar object.
I'm passing it here down for convenience so that the lib user doesn't has to check the prop names he uses.